packages/editor/src/lib/components/default-components/DefaultInFrontOfTheCanvas.tsx

```diff
@@ -0,0 +1,40 @@
+import { useValue } from '@tldraw/state-react'
+import { useEffect } from 'react'
+import { useEditor } from '../../hooks/useEditor'
+
+export const DefaultInFrontOfTheCanvas = () => {
```

Comment:
(just passing by here, but given this is very often overridden by devs, is there a way we can either make sure they know to still include it or preferably include it in a different place so it won't get overridden by devs trying to make on-canvas UI? Maybe we put it in `DefaultCanvas` instead?)

Reply:
yeah, fair enough!

i put it in `DefaultCanvas` — if you think it's something that should be overridable i think that could be alright too but i didn't that to start.